src/io/flutter/ProjectOpenActivity.java

```diff
@@ -41,6 +42,8 @@ public ProjectOpenActivity() {
 
   @Override
   public void runActivity(@NotNull Project project) {
+    JxBrowserManager.get().setUp(project);
```

Comment:
I think we only want to start this if we know this is a flutter project; I'd move this down somewhere below the `declaresFlutter` check.